addOns/authhelper/src/main/zapHomeFiles/reports/auth-report-json/report.json

```diff
@@ -11,6 +11,14 @@
 			"key": [[${sumItem.key}]]
 		}[/th:block]
 	]
+	[#th:block th:if="${rptData.hasFailureDetails()}"]
+	,"failureReasons": [[#th:block th:each="failureItem, state: ${rptData.getFailureDetails()}"]
+		{
+			"key": [[${failureItem.getKey()}]],
+			"description": [[${failureItem.getDetail()}]]
+		}[#th:block th:if="${!state.last}"],[/th:block][/th:block]
```

Comment:
Using the same construct as the other loops would be better to keep consistency.